meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -79,16 +81,28 @@ cur_counter=0
 if [ -f "/mnt/sysroot/active/counter" ]; then
 	cur_counter=$(cat /mnt/sysroot/active/counter)
 fi
-echo $(($cur_counter + 1)) > "$SYSROOT/counter.new"
+echo $((cur_counter + 1)) > "$SYSROOT/counter.new"
 sync -f "$SYSROOT"
 mv "$SYSROOT/counter.new" "$SYSROOT/counter"
 sync -f "$SYSROOT"
```

Comment:
All the steps between `# Mark it as current hostapp` on line 73 and here should be run after the hooks and only in the case they were successful. Otherwise no modifications to the current hostapp should be written to disk

Reply:
Done. Will push with next iteration.